pkg/app/rds_aurora_mysql.go

```diff
@@ -205,7 +206,7 @@ func (a *RDSAuroraMySQLDB) Install(ctx context.Context, namespace string) error
 	}
 
 	// create db instance in the cluster
-	_, err = rdsCli.CreateDBInstanceInCluster(ctx, a.id, fmt.Sprintf("%s-instance-1", a.id), AuroraDBInstanceClass, string(function.DBEngineAuroraMySQL), a.dbSubnetGroup)
+	_, err = rdsCli.CreateDBInstanceInClusterForTest(ctx, a.id, fmt.Sprintf("%s-instance-1", a.id), AuroraDBInstanceClass, string(function.DBEngineAuroraMySQL), a.dbSubnetGroup, a.publicAccess)
```

Comment:
I think we should explicitly set `a.publicAccess` to `false` to make it clear that this value that we are passing here is false.

Reply:
Sorry, I didn't understand clearly.
So now if we explicitly set the value to be `false`, are you saying we will not be able to change that later? Because that is already happening according to me, the value is already set to false, I suggested that we should make that explicit so that someone who reads the code would understand that by default its `false`.